### PR TITLE
added faucet funding.

### PR DESCRIPTION
**Pull request by StefanIliev545**
### Why this change is needed

Please provide a description and a link to the underlying ticket

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/obscuronet/obscuro-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

File: contracts/deployment_scripts/funding/layer1/002_fund_faucet.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    
+    const {deployer} = await hre.getNamedAccounts();
+    const l1Accs = await layer1.getNamedAccounts();
+    
+    const messageBusAddress = process.env.MESSAGE_BUS_ADDRESS!!
+    const prefundAmountStr = process.env.PREFUND_FAUCET_AMOUNT!!
+
+    if (prefundAmountStr == "0") {
+        return;
+    }
```
Comment (otherview) [inline contracts/deployment_scripts/funding/layer1/002_fund_faucet.ts:16]: It should error out here to provide some feedback

**Review by otherview — APPROVED** [review]
Just missing adding the error log, lgtm otherwise

**Review by BedrockSquirrel — CHANGES_REQUESTED** [review]
`PREFUND_FAUCET_AMOUNT` doesn't get set does it?

**Review by BedrockSquirrel — APPROVED** [review]
LGTM

**Pull request merged**